theme-bases.scss

```
@@ -0,0 +1,153 @@
+@use '@angular/material' as mat;
+@use '@brightlayer-ui/colors/palette.scss' as blui;
+
+@use './darkTheme';
+@use './blueTheme';
+@use './blui-component-theme';
+@use './margins';
+@use './fonts';
+@use './typography';
+@use './common';
+
+@mixin shareConfig() {
```

Extremely minor, but can we name this mixin "sharedConfig" instead
```suggestion
@mixin sharedConfig() {
```